eli5/sklearn/permutation_importance.py

```diff
@@ -15,7 +15,10 @@
 from sklearn.metrics.scorer import check_scoring  # type: ignore
 
 from eli5.permutation_importance import get_score_importances
+from eli5.sklearn.utils import pandas_available
 
+if pandas_available:
+    import pandas as pd
```

Comment:
Looks like the build failure https://travis-ci.org/TeamHG-Memex/eli5/jobs/492103801#L781 is due to missing   ``# type: ignore`` commend at the end here:
```suggestion
    import pandas as pd   # type: ignore
```